core/controllers/editor.py

```diff
@@ -101,9 +101,53 @@ def _require_valid_version(version_from_payload, exploration_version):
             % (exploration_version, version_from_payload))
 
 
+class EditorLogoutHandler(base.BaseHandler):
+    """Handles logout from editor page."""
+
+    @acl_decorators.can_play_exploration
+    def is_valid_exploration_id(self, exp_id):
```

Comment:
Decorators are only meant for get(), put(), etc. Drop this method altogether.

Reply:
ok. removed it